### PR TITLE
Optimize czech pulsar configuration

### PR DESCRIPTION
**Pull request by martindemko**

### CODE REVIEW

File: files/galaxy/tpv/destinations.yml.j2 [hunk 1]
```diff
@@ -244,12 +244,13 @@ destinations:
...
     max_accepted_cores: 32
-    max_accepted_mem: 128
+    max_accepted_mem: 256
     min_accepted_gpus: 0
     max_accepted_gpus: 0
     env:
       LC_ALL: C
-      SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      #SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      SINGULARITY_CACHEDIR: "/cvmfs/singularity.galaxyproject.org/all/"
```
Comment (sanjaysrikakulam) [inline files/galaxy/tpv/destinations.yml.j2:253]: I don't think you can use the CVMFS as your Singularity cache directory. Singularity will usually download the images to this directory, and you would need write access to it. Please correct me if I am wrong.

File: files/galaxy/tpv/destinations.yml.j2 [hunk 2]
```diff
@@ -273,7 +275,8 @@ destinations:
     max_accepted_gpus: 1
     env:
       LC_ALL: C
-      SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      #SINGULARITY_CACHEDIR: "/storage/praha5-elixir/home/galaxyeu/singularity"
+      SINGULARITY_CACHEDIR: "/cvmfs/singularity.galaxyproject.org/all/"
```
Comment (sanjaysrikakulam) [inline files/galaxy/tpv/destinations.yml.j2:279]: Same comment as above

**Comment by martindemko** [comment]
Maybe I got it wrong, but how is it different from the cache_directory parameter in container_resolvers in galaxy.yml? Cause there it needs to be in order to look for images in CVMFS instead of the biocontainers.

**Comment by sanjaysrikakulam** [comment]
Right, sorry for the confusion. You are correct; we indeed configure Galaxy to use the `CVMFS` for `cached_mulled_singularity cache_directory` in the container resolvers conf.

**Review by sanjaysrikakulam — APPROVED** [review]
Let's give this a try.

**Comment by martindemko** [comment]
Thanks ;)

**Comment by martindemko** [comment]
So, I think, there is something wrong. At least the alfofold says:
Unbale to run job due to a misconfiguration of Galaxy job running system

**Comment by martindemko** [comment]
I wonder, are changes already deployed?

**Comment by sanjaysrikakulam** [comment]
I deployed the changes so you can test them and got lost in work.

File: templates/galaxy/config/job_conf.yml [hunk 3]
```diff
@@ -169,18 +169,6 @@ galaxy_jobconf:
       params:
         amqp_url: "pyamqp://galaxy_cz01:{{ rabbitmq_password_galaxy_cz01 }}@mq.galaxyproject.eu:5671//pulsar/galaxy_cz01?ssl=1"
         galaxy_url: "https://usegalaxy.eu"
-        manager: production
```
Comment (sanjaysrikakulam) [inline templates/galaxy/config/job_conf.yml:172]: Maybe the job failed because `manager` was no longer specified.

**Comment by sanjaysrikakulam** [comment]
The reason was that the 

```
 Mapped job to destination id: pulsar_cz02_tpv
Invalid job runner: pulsar_eu_cz02
```

Please check your user preferences. Maybe it is still pointing to the destination that you removed. Can you please submit a PR that removes the `cz02` completely from destinations, and user_preferences_extra_conf.yml

File: templates/galaxy/config/job_conf.yml (hunk 3 above)
Comment (martindemko) [reply templates/galaxy/config/job_conf.yml:172]: if not specified, _default_ should be used which I have specified on pulsar.

**Comment by martindemko** [comment]
I have found it, thanks for your help, it helped a lot. Submiting new PR

**Pull request merged**